### PR TITLE
feature(liking functionality): Users should be able to like/dislike articles

### PR DESCRIPTION
**Pull request by Etomovich**
**Description**
<hr/>

This PR adds liking of articles functionality. It allows users to see whether they have liked/disliked an article once they  are logged in. It is also paramount that users who are not logged in not to like or dislike articles but instead they are redirected to the login modal as this PR depicts. This PR also addresses the fact that all users regardless of whether they are logged in or not should be able to view all the likes and dislikes of an articles(the totals). 

**Type of Change**
<hr/>

- [x] feature(a non-breaking change that allows users to like/dislike articles).

**Checklist**
<hr/>

- [x] Users should be able see whether they have liked/disliked an article once they log in.
- [x] Users cannot like an article once they have logged out.
- [x] Users should be able to see all article likes/dislikes.
- [x] Add test to show like functionality operations.

**Related Stories**
<hr/>

[Add the ability to like/dislike articles](https://www.pivotaltracker.com/n/projects/2313280/stories/164046218)

### CODE REVIEW

File: src/containers/GetAnArticle.jsx [hunk 1]
```diff
@@ -34,17 +144,68 @@ export class GetAnArticle extends Component {
...
+                  </i>
+                </a>
+              </div>
+            </div>
+          </div>
+          <div className="auth col-md-10">
+            <AnArticle data={article} />
+            <br />
+            <hr />
+            <br />
```
Comment (abulojoshua1) [inline src/containers/GetAnArticle.jsx]: Is it possible to use CSS for this?

File: src/tests/reducers/likeArticleReducer.test.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+            })
+        ).toEqual(
+            {
+              hasLiked: false,
+              likeState: false,
+              likeID: -1,
+            },
+        )
+      })
+});
```
Comment (abulojoshua1) [inline src/tests/reducers/likeArticleReducer.test.js]: Add EOF.

**Review by abulojoshua1 — CHANGES_REQUESTED** [review]
Mostly looks good, just make minor changes.

File: src/containers/GetAnArticle.jsx (hunk 1 above)
Comment (Etomovich) [reply src/containers/GetAnArticle.jsx]: @abulojoshua1 I have removed this and used a div.

File: src/tests/reducers/likeArticleReducer.test.js (hunk 2 above)
Comment (Etomovich) [reply src/tests/reducers/likeArticleReducer.test.js]: Added this

**Review by abulojoshua1 — APPROVED** [review]
LGTM

**Pull request merged**